composer.json

```diff
@@ -30,7 +30,8 @@
         "m6web/coke": "~1.2",
         "m6web/symfony2-coding-standard": "~1.2",
         "symfony/console": "^4.4 || ^5.0",
-        "symfony/http-foundation": "^4.4 || ^5.0"
+        "symfony/http-foundation": "^4.4 || ^5.0",
+        "symfony/yaml": "^4.4 || ^5.0"
```

Comment:
Can you move this two line in `require` section ?

You should also add `symfony/validator`